obspy/io/mseed/core.py

```diff
@@ -495,6 +495,12 @@ def allocate_data(samplecount, sampletype):
     del lil  # NOQA
     return Stream(traces=traces)
 
+# Define helper function to copy data, replaces use of: trace.data.copy().astype(dtype)
+def _np_copy_astype(data, dtype):
+    try:
+        return data.astype(dtype, _np_copy_astype=True)
```

Comment:
this needs to be `..., copy=True)`

Reply:
oops, that must have auto-filled in and I didn't notice, thanks!

Reply:
well, we'd have seen it in failing tests anyway.. that's what they're there for anyway ;-)